spec/services/proofing/aamva/proofer_spec.rb

```diff
@@ -44,6 +44,307 @@
   end
 
   describe '#proof' do
+    describe 'individual attributes' do
+      subject(:result) do
+        described_class.new(AamvaFixtures.example_config.to_h).proof(state_id_data)
+      end
+
+      def self.when_missing(&block)
+        context 'when missing' do
+          let(:verification_response) do
+            XmlHelper.delete_xml_at_xpath(
+              AamvaFixtures.verification_response,
+              "//#{match_indicator_name}",
+            )
+          end
+
+          instance_eval(&block)
+        end
+      end
+
+      def self.when_unverified(&block)
+        context 'when unverified' do
+          let(:verification_response) do
+            XmlHelper.modify_xml_at_xpath(
+              AamvaFixtures.verification_response,
+              "//#{match_indicator_name}",
+              'false',
+            )
+          end
+
+          instance_eval(&block)
+        end
+      end
+
+      def self.test_in_requested_attributes(logged_attribute = nil)
+        if logged_attribute
+          it "does not stop #{logged_attribute} from appearing in requested_attributes" do
+            expect(result.requested_attributes).to include(logged_attribute => 1)
+          end
+          it 'does not itself appear in requested_attributes' do
+            expect(result.requested_attributes).not_to include(attribute => 1)
+          end
+        else
+          it 'appears in requested_attributes' do
+            expect(result.requested_attributes).to include(attribute => 1)
+          end
+        end
+      end
+
+      def self.test_not_in_requested_attributes(logged_attribute = nil)
+        if logged_attribute
+          it "stops #{logged_attribute} from appearing in requested_attributes" do
+            expect(result.requested_attributes).not_to include(logged_attribute => 1)
+          end
+        end
+        it 'does not appear in requested_attributes' do
+          expect(result.requested_attributes).not_to include(attribute => 1)
+        end
+      end
+
+      def self.test_in_verified_attributes(logged_attribute = nil)
+        if logged_attribute
+          it "does not stop #{logged_attribute} from appearing in verified_attributes" do
+            expect(result.verified_attributes).to include(logged_attribute)
+          end
+          it 'does not itself appear in verified_attributes' do
+            expect(result.verified_attributes).not_to include(attribute)
+          end
+        else
+          it 'appears in verified_attributes' do
+            expect(result.verified_attributes).to include(attribute)
+          end
+        end
```

Comment:
Looks like for all the times we actually use this method, it has `logged_attribute`, so no need to have the branching, right?

Or is this planning ahead for the other branch
```suggestion
      def self.test_in_verified_attributes(logged_attribute)
        it "does not stop #{logged_attribute} from appearing in verified_attributes" do
          expect(result.verified_attributes).to include(logged_attribute)
        end

        it 'does not itself appear in verified_attributes' do
          expect(result.verified_attributes).not_to include(attribute)
        end
```